### ISSUE TITLE
A11y_mssql for VSCode_mssql Extension_Connections_Keyboard: Tooltip present on connections and query history is not accessible with keyboard.

### ISSUE THREAD

**Issue by aniketr99** [microsoft/vscode-mssql#17879]
## GitHubTags:
#A11yTCS;#Win11;#DesktopApp;#BM_mssql for VSCode_Win32_Feb2024;#A11ySev2;#mssql for VSCode;#WCAG2.1.1;#A11yMAS;#Win32;#AINotInScope;#Keyboard;A11yAuto;

## Environment Details:
Application Name: mssql for VSCode
Operating System: Win11[23H2]
OS Build: 26016.1012
Visual studio code insider: 1.86.2
Extension Version: v1.22.1

## Repro Steps:

1. Open Visual studio code 
2. Click on extension button and search for "mssql"
3. Click on install and move to the SQL Server  button
4. Click on Add Connection Button and insert the valid credentials
5. Verify that all the elements of "New Connection "screen are accessible and MAS Compliant.
6. Tab till Connections section in the left side and verify whether tooltip is accessible with keyboard or not.

## Actual:
Tooltip present on connections and query history is not accessible with keyboard.

### Note:
Similar issue is also observed on query history section as well.

## Expected:
Tooltip present on connections and query history should be accessible with keyboard.

## User Impact:
Keyboard users will not get about the controls if Tooltips are not appearing when controls are receiving focus.

[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

## Attachment:
![Presentation5](https://github.com/microsoft/vscode-mssql/assets/100269580/c75e27c3-208a-45f7-a224-776379b90472)

**Comment by alanrenmsft** [comment]
@aniketr99 please refer to https://github.com/microsoft/vscode-mssql/issues/17201 and downgrade this one to sev3. thanks.

**Comment by aniketr99** [comment]
As per above comment, downgrading the severity to sev3. Thanks

**Comment by laurenastrid1** [comment]
Closing as this is a duplicate issue of https://github.com/microsoft/vscode-mssql/issues/17201
